PennMobile/Events/EventsAPI.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EventsAPI.swift
+//  PennMobile
+//
+//  Created by Samantha Su on 10/1/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import SwiftyJSON
+import Foundation
+
+class EventsAPI: Requestable {
+    static let instance = EventsAPI()
+    
+    let eventsUrl = "https://penntoday.upenn.edu/events-feed?_format=json"
+    
+    func fetchEvents(_ completion: @escaping (_ result: Result<[PennEvents], NetworkingError>) -> Void) {
+        getRequestData(url: eventsUrl) { (data, error, statusCode) in
+            if statusCode == nil {
+                return completion(.failure(.noInternet))
+            }
+            
+            if statusCode != 200 {
+                return completion(.failure(.serverError))
+            }
+            
+            guard let data = data else { return completion(.failure(.other)) }
+            
+            let decoder = JSONDecoder()
+            
+            let formatter = DateFormatter()
+           formatter.locale = Locale(identifier: "en_US_POSIX")
+           formatter.timeZone = TimeZone(abbreviation: "EST")
+           formatter.dateFormat = "MM/dd/yyyy"
+            
+            decoder.dateDecodingStrategy = .formatted(formatter)
+            
+            do {
```

Comment:
Rather than using a do-try block, lets use a `if let events = try? decoder..` and an else block that return `completion(.failure(.serverError))`  instead of printing the error statement